src/components/Errors/Alert.tsx

```diff
@@ -9,6 +9,52 @@ const Alert: React.FC<AlertProps> = ({ message, dispatch }) => {
   const dismissError = () => {
     dispatch({ type: 'DISMISS_ERROR', payload: null });
   };
+
+  const renderMessage = () => {
+    return message === `Sorry! We currently don't support WASI.` ? (
```

Comment:
Could we specify further to say that `Sorry, WASI is currently not supported in the Playground. For now, use the Extism CLI to test WASI plug-ins.`

Reply:
Sure, will do.